docs/planning/timeline-editing.mdx

```diff
@@ -40,40 +40,94 @@ Click on the plus button to add a new row to the timeline. A row will be added b
   </figcaption>
 </figure>
 
 ### Add an Activity Layer
 
-To visualize activities on the timeline add a new layer using the plus button in the Layers section. By default new layers have a layer type of Activity. Additionally, the default Aerie UI view has a single row with a single activity layer with all activities visualized by default. Manage which activities are visualized in an activity layer by clicking on the search input for the layer and selecting activities. Activities can be filtered by substring match using the input. Once you have a filter applied you can also select all matches using the Select X activities button at the bottom of the result list. You can also manage the color of the layer using the color picker. Below is a demonstration of this workflow.
+To visualize activities on the timeline add a new layer using the plus button in the Layers section. By default new layers have a layer type of Activity. Additionally, the default Aerie UI view creates a row with a single activity layer with all activities grouped by type. Manage which activities are visualized in an activity layer by clicking on the search input for the layer and selecting activities. Activities can be filtered by substring match using the input. Once you have a filter applied you can also select all matches using the Select X activities button at the bottom of the result list. You can also manage the color of the layer using the color picker. Below is a demonstration of this workflow.
 
 <video controls>
   <source src={timelineActivityEditingDemo} />
 </video>
 
-### Managing Activity Layer Height
-
-Rows by default have a manually specified height. The UI will pack directives and simulated activities in a layer to make the most efficient use of the available space. However if the packed content is still too big for the manually specified height the content will begin to overdraw upon itself. To resolve this issue you can input a larger manual height for the row or specify that the row automatically calculate the minimum height needed to prevent overdraw. Control the row height manually from within the Details section of the timeline row editor or by dragging on the bottom of the row. Alternatively use the Auto option from within the Resize Mode dropdown in the Details section to have the row automatically resize to the content.
+<br />
+<br />
 
-:::info Note
-Use of Auto mode can result in degraded Aerie UI performance when visualizing thousands of activities on the timeline. It is recommended that you use a smaller, manually specified height for this scenario. Alternatively you can filter the activities down to a more manageable set to improve performance.
+:::info
+Note All activity layers within a row are drawn on the same canvas so that all of the activities are grouped and packed together correctly. Activities will take on the color of the layer that they came from.
```

Comment:
Keeping the `Note` up top sets the title of the box to `Note` while using the `info` icon. By moving it onto a new line, the callout renders incorrectly.

<img width="283" alt="image" src="https://github.com/NASA-AMMOS/aerie-docs/assets/62574120/1380042d-b678-4fb4-9b84-2048b5d9a9fb">


Additionally, I'm not sure how much the average user will understand this note? Heck, _I_ don't really understand this note. Is it saying that regardless of if an activity is rendered within the subfolder of another one (ie via a decomp), it'll keep its coloring?

```suggestion
:::info Note 
All activity layers within a row are drawn on the same canvas so that all of the activities are grouped and packed together correctly. Activities will take on the color of the layer that they came from.
```

Reply:
Good catch on the `Note` bit. And fair, maybe I'll remove this note and just add the first part of it to the general description of activity layers. But yes, basically we're in an awkward design stage where activities in a row are represented by multiple layers even though they all end up on the same canvas but you can still independently color each layer.